app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,13 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <permission
+        android:description="@string/permissionReadCardsDescription"
+        android:icon="@drawable/ic_launcher_foreground"
+        android:label="@string/permissionReadCardsLabel"
+        android:name="me.hackerchick.catima.READ_CARDS"
```

Comment:
> but IIRC this could cause installation conflicts from the regular and debug builds, not allowing both at the same time?

That only happens if there's a conflict between content provider authorities. In the AndroidManifest I set the authority to `${applicationId}.contentprovider.cards`, so both channels can be installed in parallel.

I did forget about the permission. Just tested it, and granting it once on either channel grants it on both apps. Do you think this is OK?

Reply:
Sounds fine to me, yeah